src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -58,20 +64,21 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get written feedback.
-    addTermRating(request);
+    addTermRating(request, /* DatastoreService */ db);
     response.setContentType("text/html; charset=UTF-8");
     response.setCharacterEncoding("UTF-8");
   }
 
-  public void addTermRating(HttpServletRequest request) throws IOException {
-    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  public void addTermRating(HttpServletRequest request, DatastoreService db) throws IOException {
     StringBuilder stringBuilder = new StringBuilder();
     String line = null;
+    // Talked with Luke and as we won't do validation testing, I will not
+    // test either of the try-catch loops.
     try {
       BufferedReader reader = request.getReader();
       while ((line = reader.readLine()) != null) stringBuilder.append(line);
     } catch (Exception exception) {
-      /*report an error*/
+      // If it could not read request.
```

Comment:
I feel like the error message you put in here makes this pretty comment a bit redundant